inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sinks/SenderManager.java

```diff
@@ -206,7 +206,13 @@ public void onMessageAck(SendResult result) {
                 return;
             }
             metric.incSendSuccessNum(bodyList.size());
-            taskPositionManager.updateSinkPosition(jobId, sourcePath, bodyList.size());
+            /**
```

Comment:
It's not necessary to use Javadoc in any method.

Reply:
It's confusing if I don't add this comment